stock_quantity_history_date/models/date_range.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Quartile.co
+#   (http://www.quartile.co)
+
+from odoo import fields, models
+
+
+class DateRange(models.Model):
+    _inherit = "date.range"
+
+    inventory_valuate_date = fields.Datetime(string="Inventory Valuate Date")
```

Comment:
```suggestion
    valuation_date = fields.Datetime(
        string="Inventory Valuate Date",
        help="Set the date that is intended to be used for showing the inventory at date."
    )
```
Translation of the help: 時点在庫出力で使用すべき日時を設定してください。

Reply:
Thank you. I added help and translation.